### ISSUE TITLE
pilot (e2e) tests take longer than 20 mins and get aborted

### ISSUE THREAD

**Issue by ldemailly** [istio/istio#3647]
https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/3599/istio-pilot-e2e/3263/
```
I0221 09:00:32.616] FAIL	istio.io/istio/tests/e2e/tests/pilot	1200.223s
I0221 09:00:32.695] tests/istio.mk:90: recipe for target 'e2e_pilot' failed
W0221 09:00:32.796] panic: test timed out after 20m0s
```

**Comment by nmittler** [comment]
@ldemailly is this still an issue after we [bumped up the prow environment to 4 machines](https://github.com/istio/istio/pull/3663)?

**Comment by ldemailly** [comment]
I don't know, @douglas-reid have you seen that error this oncall?

**Comment by nmittler** [comment]
AFAIK this was fixed by #3663, which was submitted after this issue was created (later on the same day). Closing for now.
